vhdbuilder/packer/test/linux-vhd-content-test.sh

```diff
@@ -347,4 +347,4 @@ testKubeBinariesPresent $1
 testKubeProxyImagesPulled $1
 testImagesRetagged $1
 testCustomCAScriptExecutable
-testCustomCATimerNotStarted
+testCustomCATimerNotStarted
```

Comment:
what change is here? the last empty line is removed?
![image](https://user-images.githubusercontent.com/39974812/222832351-0e4e7d90-1056-4a2f-94ca-156ac4fddfdd.png)